paper/paper.bib

```diff
@@ -139,14 +140,12 @@ @article{ghanem2022balancing
   doi = {10.1016/j.elerap.2022.101195}
 }
 
-@article{taberna2023uncertainty,
```

Comment:
I would like to keep this citation in

Reply:
I wrote a response to this, but I wanted to ensure I wasn't missing something -- why keep this citation? Would you like to suggest removing another one?

Reply:
Its a pnas paper, so a high profile journal.

Its also from close colleagues that are highly respected in ecological economics and some related fields.